atat_internal_api.yaml

```diff
@@ -1731,7 +1731,6 @@ components:
         id:
           type: string
           readOnly: true
-      additionalProperties: false
```

Comment:
Should we revert this back too? I know we previously removed it for validation, but we are using the constructed schemas for the time being.

Reply:
That's a good catch, I missed this, going to add it back.